builds/release/templates/test-core-images.yaml

```diff
@@ -0,0 +1,200 @@
+parameters:
+  az.subscription: ''
+  dps.idScope: ''
+  kv.name: ''
+  pool.linux.arm.name: ''
+  pool.linux.name: ''
+  registry.address: ''
+  registry.username: ''
+  verbose: false
+  version.core: ''
+  version.diagnostics: ''
+
+jobs:
+- job: test
+  displayName: Test Linux
+
+  variables:
+    build.config: Release
+    os: linux
+
+  strategy:
+    matrix:
+      arm32v7:
+        pool: ${{ parameters['pool.linux.arm.name'] }}
+        # Since this pipeline is about testing the Metrics Collector Docker image for release
+        # and not our host components, we'll run the arm32v7 Docker image on an arm64v8 host
+        # OS, with arm64v8 host components, for speed and convenience.
```

Comment:
Why does this save us speed and convenience?

Reply:
Because we can use a 1ES agent (speed because it's faster, convenience because it's stateless) instead of a Raspberry Pi.